test/fun_prop.lean

```diff
@@ -202,6 +203,7 @@ A silly example that everything together works as expected
 example (f : ℝ → ℝ → (ℝ →L[ℝ] ℝ)) (hf : Continuous (fun (x,y) => f x y)) :
     Measurable fun x => (f (x / x) (x * x) 1 + x) := by fun_prop
 
+set_option linter.longLine false in
```

Comment:
Are you excepting this because of the #check?

Reply:
Yes, indeed.

Given that this file is intended as both a test and a tutorial on how to use `fun_prop`, I tried to keep the original formatting.